packages/next/src/server/lib/start-server.ts

```diff
@@ -135,6 +135,18 @@ export async function startServer({
       res.end('Internal Server Error')
       Log.error(`Failed to handle request for ${req.url}`)
       console.error(err)
+    } finally {
+      if (isDev) {
+        if (
+          v8.getHeapStatistics().used_heap_size >
+          0.8 * v8.getHeapStatistics().heap_size_limit
```

Comment:
```suggestion
          0.9 * v8.getHeapStatistics().heap_size_limit
```

Reply:
Seems this used to be `90%` before it got removed https://github.com/vercel/next.js/pull/54813/files#diff-076d21e31ff35470aa69c6e887fc3f1c187648f158ee35f3fdba785d431b25b6R22-R23